src/display-buffer.coffee

```diff
@@ -860,17 +863,19 @@ class DisplayBuffer extends Model
   # Returns `null` if a wrap wouldn't occur.
   findWrapColumn: (line, softWrapColumn=@getSoftWrapColumn()) ->
```

Comment:
This method seems a bit Feature Envy (it deals mostly with another class internal). Shall we move it into `TokenizedLine`?